README.md

```diff
@@ -15,78 +18,92 @@ SilverStripe 3.0 or higher
 
 ## How to use
 
-Once you have this module installed, you'll need to configure named sets of image sizes in your sites yaml config (eg. `mysite/_config/config.yml`).
+Once you have this module installed, you’ll need to configure named sets of image sizes in your site’s yaml config (eg. `mysite/_config/config.yml`).
 Note that there are no default image sets, but you can copy the config below to get started:
 
 ```
 ---
 After: 'silverstripe-responsive-images/*'
 ---
-ResponsiveImageExtension:
+Heyday\SilverStripeResponsiveImages\ResponsiveImageExtension:
   sets:
     ResponsiveSet1:
       sizes:
-        - {query: "(min-width: 1200px)", size: 800}
-        - {query: "(min-width: 800px)", size: 400}
-        - {query: "(min-width: 200px)", size: 100}
+        '(min-width: 1200px)': [800]
+        '(min-width: 800px)': [400]
+        '(min-width: 200px)': [100]
     ResponsiveSet2:
       method: CroppedImage
       sizes:
```

Comment:
@kinglozzer I just had a fiddle with the naming of this, and `arguments` works the best in my opinion since it describes the bit that needs describing. The media query is clear, but an array of numbers is kind of ambiguous.